src/main/java/io/jenkins/plugins/gitlabserver/credentials/PersonalAccessTokenImpl.java

```diff
@@ -60,9 +61,10 @@ public Secret getToken() {
      * Our descriptor.
      */
     @Extension
+    @Symbol("gitlabPersonalAccessToken")
```

Comment:
What is the purpose of adding this symbol?

Reply:
The @ symbol denotes a Java Annotation. What a Java annotation does, is that it adds a special attribute to the variable, method, class, interface, or other language elements. (This can be configured when you declare the annotation) When you add an annotation to something, other parts of the program can check whether something has an annotation or not. It then can use this information to do whatever stuff they need.

Reply:
Thanks @markyjackson-taulia, I get it now. This has been used for the configuration as code.

Reply:
@baymac This particular annotation `@Symbol` comes from the [Structs Plugin](https://wiki.jenkins.io/display/JENKINS/Structs+plugin) and makes this accessible for JCasC and other DSL plugins (e.g. Job DSL, Jenkins Pipelines).